test/ja_serializer/builder/included_test.exs

```diff
@@ -315,4 +315,14 @@ defmodule JaSerializer.Builder.IncludedTest do
     ids = Enum.map(json["included"], &(Map.get(&1, "id")))
     assert not "p1" in ids
   end
+
+  test "non-existant include that is serialized into resource identifier results in no includes being added" do
+    c1 = %TestModel.Comment{id: "c1", body: "c1", author: "p1"}
+    a1 = %TestModel.Article{id: "a1", title: "a1", author: "p1", comments: [c1]}
```

Comment:
Are both the comment and article needed to demonstrate the issue?

Reply:
The comment may not be required